symplyphysics/laws/kinematic/vector/linear_displacement_is_angular_displacement_cross_radius.py

```diff
@@ -0,0 +1,33 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    validate_input,
+    validate_output,
+    Vector,
+    QuantityVector,
+    cross_cartesian_vectors,
+)
+
+# Description
+## Assuming a body rotating about a fixed axis, the vector of its linear displacement can be expressed
+## as the cross product of the vector of angular displacement and the radius of rotation.
+
+# Law: s = [theta, r]
+## s - vector of linear displacement
+## theta - pseudovector of angular displacement, parallel to axis of rotation
+## r - radius vector of body, perpendicular to axis of rotation
```

Comment:
Add cross product symbol here.

Reply:
We used this notation at university, and I found some references to it in other sourses. I mostly wanted to avoid non-ASCII characters in code. But I figure your intention is to use a more internationally-friendly notation, am I right?

Reply:
No, I simply want it explained. Imagine someone looking at square brackets - we need to give him guidelines, how to read it.